### ISSUE TITLE
please provide APK in the github releases and in-app updates if possible?

### ISSUE THREAD

**Issue by ASHIT-AXAR** [berty/berty#4755]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Berty product

Mobile app

### Feature request

please provide APK file in the github releases or in the F-Droid.
and in-app updates. (i mean download the APK file within the App and when it finished downloading i will update it myself, like what Telegram messenger does [the version you download from their website which has more features, not the one presented in the google play] )

### Context

 because Google Play is very hard to access in heavily censored countries.

**Comment by LFGaming** [comment]
On the github actions you can click android and then at the bottom (if you are logged in on github) you can find a artifact which is a precompiled android apk.
For more information please look at: [This github issue](https://github.com/berty/berty/issues/4552)

As for the in app update idea, that sound good I guess, I'm not entirely sure how it goes but sounds good

**Comment by jefft0** [comment]
Could be a useful feature. Keep this open as a tracking issue. Put in backlog until we have time to investigate.

**Comment by hhbisheng** [comment]
Yes, I need this. I cannot use Google Store and many people in different regions are like me

**Comment by hhbisheng** [comment]
> 你可以点击安卓操作,然后在底部点击安卓(如果你登录到了GIT),你可以找到一个工件,这是一个预先编译的安卓APK。 欲了解更多信息,请参阅:[ 这个吉塔布问题](https://github.com/berty/berty/issues/4552)
> 
> 至于应用程序更新的想法,我想这听起来不错,我不完全确定会怎样,但听起来不错。



> 你可以点击安卓操作,然后在底部点击安卓(如果你登录到了GIT),你可以找到一个工件,这是一个预先编译的安卓APK。 欲了解更多信息,请参阅:[ 这个吉塔布问题](https://github.com/berty/berty/issues/4552)
> 
> 至于应用程序更新的想法,我想这听起来不错,我不完全确定会怎样,但听起来不错。

https://github.com/berty/berty/actions/runs/6101994113
Why is this installation package so strong that it weighs 256MB? I was deeply shocked.

**Comment by hhbisheng** [comment]
I can honestly say that I am here, very close to regulation, far from Google.

**Comment by jefft0** [comment]
For more discussion of F-Droid, see https://github.com/berty/berty/issues/3101 .
